csi: support volumegroup snapshot (#13832)
rook/rook: Documentation/Helm-Charts/operator-chart.md

| `csi.enablePluginSelinuxHostMount` | Enable Host mount for `/etc/selinux` directory for Ceph CSI nodeplugins | `false` |
| `csi.enableRBDSnapshotter` | Enable Snapshotter in RBD provisioner pod | `true` |
| `csi.enableRbdDriver` | Enable Ceph CSI RBD driver | `true` |

> | `csi.enableVolumeGroupSnapshot` | Enable volume group snapshot feature. This feature is enabled by default only when the necessary CRDs are available in the cluster. | `true` |

Reviewer: I'm a little confused by the wording here including "this feature is enabled by default." Would it be more correct to say that this feature is allowed when set to true but will only be fully enabled if the necessary CRDs are available in the cluster?

Author: What i meant to convey is that the volumegroup snapshot feature will be enabled by default if the requirements crd's are present even if the value is set to `true` explicitly. Am open to suggestion on wording, Please suggest what make clear :)

Reviewer 2: ```suggestion
| `csi.enableVolumeGroupSnapshot` | Enable volume group snapshot feature. This feature is enabled by default as long as the necessary CRDs are available in the cluster. | `true` |
```
